components/menu/test/menu-rtl.visual-diff.js

```diff
@@ -56,12 +51,7 @@ describe('d2l-menu rtl', () => {
 
 		it('opens long menu item submenu on click', async function() {
 			// this scenario also tests height change going from 3 menu items to 2 within nested menu
-			await page.$eval('#nested-item-long', item => {
-				return new Promise((resolve) => {
-					item.addEventListener('d2l-hierarchical-view-show-complete', resolve, { once: true });
-					item.click();
-				});
-			});
+			await page.$eval('#nested-item-long', item => item.click());
```

Comment:
This is an example where the test is nicely simplified, and avoiding the underlying hierarchical-view event is nice.  However, I can't always explain why it's not necessary to wait a frame before taking a screenshot (which is required in the next case with the custom nested view).

Reply:
Based on my CI failure... the need to wait a frame may actually be more common.

Reply:
So waiting for the event I guess is necessary. Could this pattern be used though to simplify?

```
setTimeout(() => item.click());
await oneEvent('d2l-hierarchical-view-show-complete', item);
```